### DISCUSSION TITLE
Δημιουργία Ομάδας για το εξάμηνο 2024

### DISCUSSION THREAD

**Discussion by YGNT7777** [courses-ionio/hci#2000] [Show and tell]
Καλησπέρα συμφοιτητές,

Ανοίγω αυτή την συζήτηση, μετά απο το πρώτο μάθημα, για την δημιουργία ομάδας.
Οποίος ενδιαφέρεται να γνωριστούμε και να συνεργαστούμε στο μάθημα αυτό,
να απαντήση κάτω απο το σχόλιο (που θα έχω γραψει το ονοματεπώνυμο μου με την ΑΜ μου) 
με το ονοματεπώνυμο σου και το ΑΜ σου.

Για τώρα δεν έχω βάλει όνομα όμαδας. 
Θέλω για αρχή να γνωριστούμε. (Γενικά για το όνομα της ομάδας και για τα σχετικά  θέλω να το συζητήσουμε όλοι μαζί)

Οποίος θελεί να εποικοινωνήσει μαζί μου, να με προσθέση στο discord : natsos
Αν δεν έχεις discord κανένα πρόβλημα, απάντα απο κάτω για το πώς σε βολέυει να εποικοινωνήσουμε
θα βρούμε λύση

**Comment by YGNT7777** [comment]
ΝΙΚΟΛΑΣ ΝΑΤΣΟΣ

| **AM** |
|--------|
| 2023144|

(Δεν είναι αναγκαίο να φτιάξεις πίνακα)

**Reply by VasilisG11** [reply]
Καλησπέρα, Γκουλιώνης Βασίλειος

ΑΜ: 2019024

Σου έστειλα στο discord αίτημα για να τα πούμε εκεί (PITSIOY).

**Comment by AkisThUni** [comment]
Καλημέρα, ΘΩΔΗΣ ΠΑΤΑΠΙΟΣ

ΑΜ: 2023062

**Reply by VasilisG11** [reply]
Καλησπέρα Θώδη, άφησε μας το Discord σου να σε καλέσουμε στον σέρβερ για να γνωριστούμε και να οργανωθούμε. 
Αν δεν έχεις Discord, άσε κάποιο άλλο στοιχείο επικοινωνίας.

**Comment by GiannisBab** [comment]
ΙΩΑΝΝΗΣ ΜΠΑΜΠΛΕΚΗΣ
inf2021153

**Comment by Zardas-s** [comment]
ΣΠΥΡΙΔΩΝ ΖΑΡΔΑΣ
inf2023053

**Comment by p18tili** [comment]
Καλησπερα, ΤΗΛΙΓΑΔΗΣ ΝΙΚΟΛΑΟΣ-ΡΑΦΑΗΛ
Π2018009

**Reply by YGNT7777** [reply]
Καλησπέρα @p18tili 
πρόσθεσε με στο discord, αν δεν έχεις discord πές μου πως μπορούμε να εποικοινωνήσουμε
discord : natsos
